updatesnap/updatesnapyaml.py

```diff
@@ -47,7 +47,11 @@ def get_yaml_file(self, project_url):
         try:
             data = self._github.get_file(project_url, yaml_path)
         except (ValueError, ConnectionError):
-            data = None
+            yaml_path = 'rockcraft.yaml'
```

Comment:
I think that is better to follow the same idea that is used just after this. Something like:

```
if not data:
    yaml_path = 'rockcraft.yaml'
    try:
        data = self._github.get_file(project_url, yaml_path)
    except (ValueError, ConnectionError):
        data = None
```
and thus avoid nesting try/except blocks.